site/content/docs/5.1/examples/sidebars/sidebars.css

```diff
@@ -17,20 +19,8 @@ main {
   overflow-y: hidden;
 }
 
-.b-example-divider {
-  flex-shrink: 0;
-  width: 1.5rem;
-  height: 100vh;
-  background-color: rgba(0, 0, 0, .1);
-  border: solid rgba(0, 0, 0, .15);
-  border-width: 1px 0;
-  box-shadow: inset 0 .5em 1.5em rgba(0, 0, 0, .1), inset 0 .125em .5em rgba(0, 0, 0, .15);
-}
-
 .bi {
-  vertical-align: -.125em;
   pointer-events: none;
```

Comment:
This leftover could be dropped by using the pointer-events utility.

Reply:
Fixed via 045faaf67ecf06b2ef42ecb8054ff0027e1087de. I'll look into the main feedback about `@import` asap.